scripts/docker/conf.toml

```diff
@@ -0,0 +1,36 @@
+WebStaticFilesPath = "web"
+
+# List of allowed origins
+# e.g. "https://localhost:8080"
+CORSAllowedOrigins = []
+
+[AccountManager]
+Type = "unix"
+
+[AccountManager.OAuth2]
+Provider = "Google"
+RedirectURL = "http://localhost:8080/oauth2callback"
+
+[SecretManager]
+Type = ""
+
+[SecretManager.UNIX]
```

Comment:
I think this is not needed, you are using the empty Secret Manager Type: https://github.com/google/cloud-android-orchestration/blob/main/pkg/app/secrets/empty.go so there's no reason to define `SecretFilePath`. do you have any specific error if you omit this entry?

Reply:
No, I just had no idea about treating `SecretManager`, since existing one(`/conf.toml`) does. Removed `secrets.json`.

By the way, what's the purpose for keeping conf.toml in the root directory of this repository?